src/coreclr/nativeaot/Runtime/wasm/PalRedhawkWasm.cpp

```diff
@@ -39,4 +39,101 @@ void PalGetMaximumStackBounds_SingleThreadedWasm(void** ppStackLowOut, void** pp
     *ppStackLowOut = pStackLow;
     *ppStackHighOut = pStackHigh;
 }
+
+#ifdef TARGET_WASI
+// nop stubs
+
+int pthread_mutex_init(pthread_mutex_t * m, const pthread_mutexattr_t *__restrict a)
```

Comment:
Consistency: some of the function don't have parameter names, some do. Let all of them have it / none (I don't have a strong opinion which)?

Reply:
removed the names